packages/next/src/server/lib/router-utils/setup-dev-bundler.ts

```diff
@@ -181,8 +183,15 @@ async function startWatcher(opts: SetupOpts) {
     logging: nextConfig.logging !== false,
   })
 
+  const originalFetch = global.fetch
+  const resetFetch = () => {
+    global.fetch = originalFetch
```

Comment:
Any reason to use `global` over `globalThis`?

They are equivalent, I know, but for consistency's sake, perhaps sticking to `globalThis` would be better?